test/test_utils/test_reporting.py

```diff
@@ -241,25 +257,6 @@ def generate_sagemaker_reports(self):
                     else:
```

Comment:
For TF-inference, would the if-else condition let us run both the local and integration tests? Or, do we just want to run for the local tests?

Reply:
should run both

Reply:
Nice, missed this condition: https://github.com/aws/deep-learning-containers/pull/3658/files#diff-04427493c68d5cc481c8f1ab68627fa06699851c510bc2d1a4bad42e4bc93050R255

Thanks!